### PR TITLE
update audit checklist

### PR DESCRIPTION
**Pull request by SunJang**
SCORE Audit checklist 업데이트했습니다.

- 삭제
Time manipulation (해당사항없음)

- 추가
Unchecked Low Level Calls
Fallback
__init__ Function
Underflow/Overflow
Vault
Reentrancy

### CODE REVIEW

File: docs/audit_checklist.md [hunk 1]
```diff
@@ -210,3 +217,139 @@ Output of pseudo random number generator can be predictable if random seed is re
 # block height is predictable.
 won = block.height % 2 == 0
 ```
+## Unchecked Low Level Calls
+In case of sending ICX by calling low level function such as 'ics.send', you should check the result of calling 'icx.send' is succeed or not. Consider using 'icx.transfer' instead or prepare compensation code for failure.
```
[the remark this answers is not in the document]
Comment (SunJang) [reply docs/audit_checklist.md]: The difference of two functions described in 'dapp_guide.md'. but i can not find where it is in github. Please let me know where it is. I will add link to it.

File: docs/audit_checklist.md [hunk 2]
```diff
@@ -210,3 +217,139 @@ Output of pseudo random number generator can be predictable if random seed is re
...
+Anybody can call 'fallback' or 'tokenFallback' function by sending ICX or token. In order to your SCORE secured in your hand, Do not change ownership of the SCORE in such functions.
+```python
+# Bad
+@payable
+def fallback(self):
+    if self.msg.value >= 0:
+        self.owner = msg.sender
+```
+
+## \_\_init\_\_ Function
```
[the remark this answers is not in the document]
Comment (SunJang) [reply docs/audit_checklist.md]: As far as I know, __init__ function is only for class initialization. on_install should call super().on_install() and on_update should call super().on_update(). And it is now on the checklist.

File: docs/audit_checklist.md [hunk 3]
```diff
@@ -210,3 +217,139 @@ Output of pseudo random number generator can be predictable if random seed is re
...
+self._refund_icx_amount[_to] += amount
+if not self.icx.send(_to, amount):
+    self._refund_icx_amount[_to] -= amount
+
+# Good
+self._refund_icx_amount[_to] += amount
+self.icx.transfer(_to, amount)
+```
+
+## Fallback
```
[the remark this answers is not in the document]
Comment (SunJang) [reply docs/audit_checklist.md]: If you think it is irrelevant, should I remove the item from the checklist?

File: docs/audit_checklist-ko.md [hunk 4]
```diff
@@ -210,22 +209,139 @@ won = block.height % 2 == 0
...
+    self._total_supply.set(self._total_supply.get() + _amount)
+
+    self.Transfer(EOA_ZERO, self.owner, _amount, b'mint')
+
+# Good  
+@external
+def mintToken(self, _amount: int):
+    if not msg.sender == self.owner:
+        self.revert('Only owner can mint token')
+    if value <= 0:
```
Comment (sink772) [inline docs/audit_checklist-ko.md]: value -> _amount

File: docs/audit_checklist.md [hunk 5]
```diff
@@ -210,3 +216,142 @@ Output of pseudo random number generator can be predictable if random seed is re
...
+    self._total_supply.set(self._total_supply.get() + _amount)
+
+    self.Transfer(EOA_ZERO, self.owner, _amount, b'mint')
+
+# Good  
+@external
+def mintToken(self, _amount: int):
+    if not msg.sender == self.owner:
+        self.revert('Only owner can mint token')
+    if value <= 0:
```
Comment (sink772) [inline docs/audit_checklist.md]: value -> _amount

File: docs/audit_checklist-ko.md [hunk 6]
```diff
@@ -144,9 +143,9 @@ def Transfer(self, _from: Address, _to: Address, _value: int, _data: bytes):
 
 @external
 def transfer(self, _to: Address, _value: int, _data: bytes = None) -> bool:
-    self._balances[self.msg.sender] -= value
-    self._balances[to_] += value
-    self.Transfer(self.msg.sender, _to, value, data)
+    self._balances[self.msg.sender] -= _value
+    self._balances[to_] += _value
```
Comment (Spl3en) [inline docs/audit_checklist-ko.md]: `to_` should be `_to`

File: docs/audit_checklist-ko.md [hunk 7]
```diff
@@ -161,7 +160,7 @@ def Transfer(self, _from: Address, _to: Address, _value: int, _data: bytes):
 @external
 def transfer(self, _to: Address, _value: int, _data: bytes = None) -> bool:
     // token transfer 하지 않음
-    self.Transfer(self.msg.sender, _to, value, data)
+    self.Transfer(self.msg.sender, _to, _value, data)
```
Comment (Spl3en) [inline docs/audit_checklist-ko.md]: `data` should be `_data`

Comment (sojinkim-icon) [reply docs/audit_checklist-ko.md]: @SunJang will you please verify all the code snippets actually running?

Comment (SunJang) [reply docs/audit_checklist-ko.md]: @sojinkim-icon My bad. I will.

**Pull request merged**